packages/app/src/renderer/app/components/offline-banner/offline-banner.component.scss

```
@@ -0,0 +1,5 @@
+@import 'src/renderer/styles/bootstrap-variables.scss';
```

Stylelint (17.12.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)